kfs-cam/src/main/java/edu/arizona/kfs/module/cam/document/AssetMaintainableImpl.java

```diff
@@ -0,0 +1,111 @@
+package edu.arizona.kfs.module.cam.document;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.kns.document.MaintenanceDocument;
+import org.kuali.rice.krad.bo.Note;
+import org.kuali.rice.krad.service.KRADServiceLocator;
+import org.kuali.rice.krad.service.NoteService;
+import org.kuali.rice.krad.util.GlobalVariables;
+
+import edu.arizona.kfs.sys.KFSConstants;
+
+@SuppressWarnings("deprecation")
```

Comment:
I don't see any instances of a deprecated class being used here, is that correct? (I.e. I deleted the suppression tag, re-compiled the class, and no new warnings or errors surface). To be clear, I don't mind suppressing warnings, but I think this case is sub-optimal for two reasons: it isn't actually suppressing anything, and the suppression is the widest scope when not necessary. If we are going to suppress warnings, I'd like to do it at the smallest scope possible. Unless the same warning is multiple line in a method, then elevate suppressing to method. If multiple methods are all suppressed, then go ahead and suppress at class level. When we go to suppress first at the class level, then we lose any benefit of the warning altogether.